docs/api.md

```diff
@@ -2105,6 +2106,8 @@ Like how `Form`s are addressed by their XML `formId`, individual `Submission`s a
 
 As of version 1.4, a `deviceId` and `userAgent` will also be returned with each submission. The client device may transmit these extra metadata when the data is submitted. If it does, those fields will be recognized and returned here for reference. Here, only the initial `deviceId` and `userAgent` will be reported. If you wish to see these metadata for any submission edits, including the most recent edit, you will need to [list the versions](/reference/submissions/submission-versions/listing-versions).
 
+As of version 2023.2, this API returns `currentVersion` that contains metadata of the most recent edit of the Submission.
```

Comment:
How about "most recent version" instead of "most recent edit"? The API will return `currentVersion` even if there hasn't been an edit.